test/parallel/test-intl-v8BreakIterator.js

```diff
@@ -3,7 +3,7 @@ const common = require('../common');
 const assert = require('assert');
 const vm = require('vm');
 
-if (typeof Intl === 'undefined')
+if (common.hasIntl === 'undefined')
```

Comment:
common.hasIntl returns boolean, it does not match 'undefined'.

Reply:
Thanks for reviewing, I fixed it to expect boolean.